### PR TITLE
Add support for Avtech RoomAlert 4E

### LINKED ISSUE #3166: AVTech RoomAlert 4E Support?
Please let me know if this isn't the correct area for this.  But I have some AVTech RoomAlert 4E temp monitors that aren't drawing temperature graphs.  Are the devices supported?

temp error

Any help you can give would be appreciated.  If you need any additional info, please let me know.

My system stats are:
system stats

### PR DESCRIPTION
**Pull request by murrant**
Clean up avtech by making a helper function so it is easier to add devices

fixes #3166

### CODE REVIEW

**Comment by Rosiak** [comment]
Why not have the function in includes/functions.php?

File: includes/discovery/sensors/temperatures/avtech.inc.php [hunk 1]
```diff
@@ -3,65 +3,195 @@
...
+        avtech_add_temperature($device, $sen1);
+
+        $sen2 = array(
+            'id'        => 3,
+            'oid'       => $device_oid.'1.2.2.1.0',
+            'descr_oid' => $device_oid.'1.2.2.3.0',
+        );
+        avtech_add_temperature($device, $sen2);
+    }
+    else if(strpos($device['sysObjectID'], '.20916.1.1') !== false) {
```
Comment (laf) [inline includes/discovery/sensors/temperatures/avtech.inc.php]: Mind changing this to elseif whilst you're updating :)

**Comment by laf** [comment]
includes/discovery/functions.inc.php is probably the better place for the function.

**Comment by murrant** [comment]
I'll make it a little more generic and add to functions.php.  

Should I attempt to make it very generic or keep it targeted at a tech?

**Comment by laf** [comment]
Actually now you've side that I'm wondering why this isnt just using the discover_sensor() function.

**Comment by murrant** [comment]
@laf: there are many avtech devices with poorly structured snmp.  Basically, they are just a list of values.  An snmpwalk returns a mangled array.

So I have to fetch each value one at a time. So, instead of doing that for 11 models with 3-24 sensors each.  I wanted a way to just specify the oids for the value, description, min, and max and have the fetch and discover_sensor() done in common code.

**Comment by laf** [comment]
Ok no probs. Then I'd say function as is just needs to go into includes/discovery/functions.inc.php

**Pull request merged**